_posts/2021-04-18-api-conversion.md

```diff
@@ -0,0 +1,86 @@
+For instance `v1beta2` supports only new array based listener configuration and new metrics configuration formats from config maps.
+`metrics` and `logging` field were also changed as well.
+For metrics configuration you should use `metricsConfig` in `spec.kafka` to reference name of config map with metrics configuration in `valueFrom` field.
+Same for `logging`, but field name remains same.
+In additional some old unused fields like `tlsSidecar` were removed as well.
+All changes in fields are described in our [documentation](https://strimzi.io/docs/operators/latest/full/deploying.html#proc-upgrade-cli-tool-files-str).
+You can also see the deprecated field in CR status as warnings.
+
+For smooth migration to new CRs and CRDs version we created `api-conversion` tool. _Note that for proper conversion you need to have installed Strimzi 0.22 with all its CRDs._
+
+### API conversion tool
+
+The tool is shipped as part of Strimzi 0.22 in [zip](https://github.com/strimzi/strimzi-kafka-operator/releases/download/0.22.1/api-conversion-0.22.1.zip) and [tar.gz](https://github.com/strimzi/strimzi-kafka-operator/releases/download/0.22.1/api-conversion-0.22.1.tar.gz) archives.
+After un-archiving, you will find there all libs needed by the tool in `libs` directory and two scripts for running the tool in folder `bin`.
+Script `api-conversion.sh` is for linux/mac users and `api-conversion.cmd` is for Windows users.
+
+The tool can operate in two modes:
+* convert your resources to `v1beta2`. It can convert resources in yaml files or resource already deployed into Kubernetes cluster.
+* convert CRDs to use `v1beta2` as a stored version and use `v1beta2` in all Strimzi custom resources.
+
+You can use the tool directly from un-archived directory as it's shown in the example bellow.
+
+![api-conversion tool help](/assets/images/posts/2021-04-18-api-conversion.png)
+
+#### convert-file
+
+`convert-file` (`cf`) command basically takes input yaml file passed via option `-f` or `--file` and convert CR to new API version and migrate deprecated fields.
+The output will be printed to _stdout_ or you can specify output file with `-o` or `--output` options.
+You can also use `--in-file` option to save changes directly to input file.
+For more info run `bin/api-conversion.sh cr --help` command.
+
+#### convert-resources
+
+`convert-resources` (`cr`) allow you to convert custom resources directly in your Kubernetes cluster.
+You can use `-a`,`--all-namespaces` or `-n`,`--namespace` options to specify namespace where the resources should be converted.
+With `-k`,`--kind` the tool will convert only a specific kind of custom resources.
+This option can be passed multiple times to convert multiple kinds.
+For instance to convert `Kafka` and `KafkaTopic` you just need to pass `--kind Kafka --kind KafkaTopic` to `cr` command.
+
+The last option is `-n`,`--name` where you can specify a name of the custom resource.
```

Comment:
Do we really have `-n` for name and `-n` for namespace?

Reply:
`-n` is just for namespace, will fix it